src/transport/SecureSessionMgr.cpp

```diff
@@ -139,14 +139,19 @@ CHIP_ERROR SecureSessionMgr::PrepareMessage(SessionHandle session, PayloadHeader
 
         packetHeader.SetMessageCounter(messageCounter);
 
-        ChipLogProgress(Inet,
-                        "Build %s message %p to 0x" ChipLogFormatX64 " of type %d and protocolId %" PRIu32
-                        " on exchange " ChipLogFormatExchangeId " with MessageCounter %" PRIu32 ".",
-                        "plaintext", &preparedMessage, ChipLogValueX64(kUndefinedNodeId), payloadHeader.GetMessageType(),
-                        payloadHeader.GetProtocolID().ToFullyQualifiedSpecForm(), ChipLogValueExchangeIdFromHeader(payloadHeader),
-                        packetHeader.GetMessageCounter());
+#if CHIP_PROGRESS_LOGGING
+        destination = kUndefinedNodeId;
+#endif // CHIP_PROGRESS_LOGGING
     }
 
+    ChipLogProgress(Inet,
```

Comment:
This must be wrapped in `#if CHIP_PROGRESS_LOGGING`

Reply:
> This must be wrapped in #if CHIP_PROGRESS_LOGGING since existence of destination is also conditional.

`ChipLogProgress` is conditionally defined based on `CHIP_PROGRESS_LOGGING` to be either something or a no-op.  I could add the extra guard here too, but it would be redundant with that....

And yes, I agree the node id logging here could use more work.